admin_guide/topics/creating-primed-images-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='creating-primed-image-cluster-auto-scaler-{context}']
+= Creating a primed image
+
+You can use Ansible playbooks to automatically create and seal a
```

Comment:
@kalexand-rh 
What does _seal_ mean here? Would a typical admin know?

Reply:
@frobware, what's a plainer way to say "seal" here?

Reply:
@kalexand-rh I am not sure what that means.

Reply:
> what's a plainer way to say "seal" here?

Maybe just drop that phrase? All we care about is creating the image.